packages/tools/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,132 @@
+{
+	"name": "@fluid-tools/app-insights-logger",
+	"version": "2.0.0-internal.4.1.0",
+	"private": true,
+	"description": "Contains a fluid logging client that sends telemetry events to Azure App Insights",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/tools/client-logger/app-insights-logger"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build:compile": "npm run build:tsc && npm run build:copy-resources",
+		"build:copy-resources": "copyfiles \"src/**/*.css\" dist --up 1",
+		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"build:full": "concurrently npm:build npm:webpack",
+		"build:full:compile": "concurrently npm:build:compile npm:webpack",
+		"build:genver": "gen-version",
+		"build:tsc": "npm run tsc",
+		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"clean": "rimraf _api-extractor-temp coverage dist nyc *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"format": "npm run prettier:fix",
+		"format-and-build": "npm run format && npm run build",
+		"format-and-compile": "npm run format && npm run build:compile",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
+		"start:test-app": "start-server-and-test start:tinylicious 7070 start:test-app:client",
+		"start:test-app:client": "webpack serve --config webpack.config.js",
+		"start:tinylicious": "tinylicious",
+		"test": "npm run test:jest",
+		"test:coverage": "npm run test:jest:coverage",
+		"test:jest": "jest --detectOpenHandles",
+		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci --reporters=default --reporters=jest-junite",
+		"tsc": "tsc",
+		"webpack": "webpack --env production",
+		"webpack:dev": "webpack --env development"
+	},
+	"dependencies": {
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/container-loader": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
```

Comment:
this package should avoid taking a dependency on implementation packages, and only reference definition packages. its also a bit odd the logger is coupled to anything but common-definitions which contains the logger. i would consider splitting this package to be just a pure logger, and if you have more utils do it elsewhere. I also think this package belongs under framework instead of tools.

Reply:
Okay, yeah right now it is coupled with the example so there are many extraneous imports. I will look into splitting this and having a conversation with my team

Reply:
This package dependencies have been updated to only have the necessary dependencies and definitions as of commit https://github.com/microsoft/FluidFramework/commit/b4036517047dfb956c2a4e88c1cb3803eb8d24aa .

In regards to moving package under  /framework, could you please give some more insight? I am unfamiliar with the package's domain

Reply:
@anthony-murphy several people in our team had the same question regarding tools/ vs framework/, including myself. I can see arguments for both but we wanted to understand why for you it belongs in X instead of Y.

Reply:
framework as pieces consumers use, tools are independently executable programs, which generally aren't shipped. debugger is a bit odd, and doesn't fit with the previous tools, so that may not be in the right place either.